core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/AbstractRun.java

```diff
@@ -81,39 +90,59 @@
   @Transient
   public Document submissionDocument;
 
-  @OneToOne(cascade = CascadeType.ALL)
+  @OneToOne(targetEntity = SecurityProfile.class, cascade = CascadeType.ALL)
   private SecurityProfile securityProfile;
 
   private String name;
+  @Column(nullable = false)
   private String alias;
   private String description;
   private String accession;
   private Integer platformRunId;
+
+  @Column(nullable = false)
   private Boolean pairedEnd;
   private Integer cycles;
   private String filePath;
 
-  private Date lastUpdated;
-
   @OneToOne(targetEntity = StatusImpl.class, cascade = CascadeType.ALL)
   private Status status;
```

Comment:
Could make this a true OneToOne by adding `Status.runId` as its new primary key and dropping `Status.statusId` and `Run.status_statusId`. If you want. :grin: